types/esbuild-plugin-ignore/commonjs/package.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This package.json seems like it might cause problems with our publisher. What is it needed for? Could you instead name the file commonjs/index.d.cts?

Reply:
This is a layout that some packages use to avoid using non .js extensions; I've never seen it used by anyone other than `tshy` (which emits TS), though.

I'm not sure how one can make a DT package for this without the extra package.jsons; we may need to come up with a publisher solution that includes these files...

Reply:
Hm, _maybe_ the idea of pointing it at a `cts` file would work, though it would be a lie...

@ashkuc I would suggest making a `test.cts` file and write `import ... = require("...")` to verify that things work.

Reply:
Yeah, I forgot, just added it. Thank you for the advices and guidance!